### PR TITLE
v5.0.0-proposal

### PR DESCRIPTION
**Pull request by khanayan123**
### Breaking Changes

More information about the breaking changes from this release can be found in
the [migration guide](https://github.com/DataDog/dd-trace-js/blob/master/MIGRATING.md).

* **core:** drop support for Node 16 (#3905)
* **core:** fix TypeScript errors (#3912)

### Features

* **profiling:** Add experimental CPU profiler (#3895)

### Improvements

* **profiling:** GA Code hotspots and endpoint collection (#3940)

### CODE REVIEW

File: packages/dd-trace/src/sampling_rule.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
+'use strict'
```
Comment (tlhunter) [inline packages/dd-trace/src/sampling_rule.js:1]: Just to confirm, is this feature not landing in the v4.x branch? @Qard @khanayan123

Comment (khanayan123) [reply packages/dd-trace/src/sampling_rule.js:1]: this feature already landed on v4.x and v3.x. This PR rebased on top of master but v5.x is behind master which is why this commit is showing up

**Comment by simon-id** [comment]
aren't we missing updates in the README ? like saying that v5 is latest instead of v4 and updating the release schedule table ?

**Comment by khanayan123** [comment]
> aren't we missing updates in the README ? like saying that v5 is latest instead of v4 and updating the release schedule table ?

Those updates are in this PR: https://github.com/DataDog/dd-trace-js/pull/3918

which will be merged in after this PR

**Pull request closed without merging**